netebpfext/net_ebpf_ext_bind.c

```diff
@@ -38,10 +38,31 @@ const net_ebpf_extension_wfp_filter_parameters_t _net_ebpf_extension_bind_wfp_fi
 
 #define NET_EBPF_BIND_FILTER_COUNT EBPF_COUNT_OF(_net_ebpf_extension_bind_wfp_filter_parameters)
 
+static uint64_t
```

Comment:
Why do we need an explicit ```uint64_t``` return type instead of some ```ebpf_xxx_t```?  We _are_ using other ```epbf_xxx_t``` typedefs so was curious why we do not have a typedef for this return type.  Also, the implementations seem to return ```NTSTATUS``` values which are 32-bit, so was curious about the need to promote them to 64 bits.

Reply:
Good quesiton, I think this ought to be ebpf_status_t.
For example, it looks like https://github.com/microsoft/ebpf-for-windows/blob/9612ee430de60e41462f6611fd1b7e5d67f0ac3c/libs/execution_context/ebpf_program.c#L1559 takes the return from context_create and stores it in an ebpf_status_t.